tools/SchemaManager/Commands/ApplyCommand.cs

```diff
@@ -61,7 +61,7 @@ private async Task HandlerAsync(string connectionString, Uri server, MutuallyExc
 
         private bool EnsureForce()
         {
-            _logger.LogWarning(Resources.ForceWarning);
+            _logger.LogWarning("Are you sure to apply command with force option? Type 'yes' to confirm.");
```

Comment:
Interesting... I think we should probably track a backlog item to clean this up. We should really separate out what we want to be written to the console in an interactive manner vs what we want to log which should likely go to a file sink in this context.